test/e2e/login_logout_test.go

```diff
@@ -113,6 +113,18 @@ var _ = Describe("Podman login and logout", func() {
 		Expect(session).To(ExitWithError())
 	})
 
+	It("podman login and logout without registry parameter", func() {
+		SkipIfRootless()
+		session := podmanTest.Podman([]string{"login", "-u", "podmantest", "-p", "test"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.LineInOutputContains("docker.io"))
+		Expect(session.ExitCode()).To(Not(Equal(0)))
```

Comment:
I think this is testing the case when there's no registry in registry.conf and it fails.  Can we test the case when there is an entry in registries.conf that woud succeed?